cli/pkg/autodetect/kind.go

```diff
@@ -22,7 +22,7 @@ var CurrentKubernetesVersion KubernetesVersion
 const (
 	KindUnknown   Kind = "Unknown"
 	KindMinikube  Kind = "Minikube"
-	KindKind      Kind = "KinD"
```

Comment:
I think this was written like so intentionally - "**K**ubernetes in **D**ocker".
@edeNFed

Reply:
Nowhere on their site, they use that form.

Reply:
I am ok with both